src/content/components/BugList/BugList.js

```diff
@@ -76,89 +80,148 @@ export class BugList extends React.PureComponent {
   }
 
   renderFilters() {
-    return (<EditorGroup>
-      {this.props.showResolvedOption ? <span><input type="checkbox" onChange={this.onCheckShowResolved} checked={this.state.showResolved} /> Show Resolved</span> : null}
-    </EditorGroup>);
+    return (
+      <EditorGroup>
+        {this.props.showResolvedOption ? (
+          <span>
+            <input
+              type="checkbox"
+              onChange={this.onCheckShowResolved}
+              checked={this.state.showResolved}
+            />{" "}
+            Show Resolved
```

Comment:
Hmm.. I guess that's one way to insert a space ?

Reply:
Ack prettier does weird stuff